packages/@react-stately/selection/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
+    "@react-aria/utils": "^3.9.0",
```

Comment:
we still can't do this, you'll need to do what i mentioned in slack and push it down into stately and re-export in aria